src/ttkbootstrap/localization/msgs/pt.msg

```diff
@@ -1,20 +1,20 @@
 namespace eval ::tk {
```

Comment:
The `namespace eval ::tk {` brackets should be removed and the remaining text pushed to the left margin. You can checkout the new msg files.

I found that `namespace eval ::tk` was preventing me from translating multi-word strings. Some quirk of moving between tcl & Python I think.

Also, would you mind submitting this one as a separate PR?  There are a lot of table updates and those will take longer to review when you've settled on the changes.